### PR TITLE
US026 - Add branding to the product

### PR DESCRIPTION
**Pull request by Brigid481**
Adding Branding to homepage

### CODE REVIEW

**Review by dylanlaffin — DISMISSED** [review]
from my experience with css everything looks gook.

**Comment by RaggedyRach** [comment]
Hi, maybe remove some spacing and tidy up the unused files. The images and the stylesheet. thx :)

**Review by RaggedyRach — APPROVED** [review]
spaces and redundant files moved. Looks good to me :)

File: views/homepage.html [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    <link rel="icon" href="/images/favicon.png">
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.7.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<style>
```
Comment (shaunganley) [inline views/homepage.html]: move this section to a css file and import above

File: views/homepage.html [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+                            alt="Kainos">
+                        <div class="card-body">
+                            <h3 class="h3"> Together we write our story...</h3>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <footer class="footer">
```
Comment (shaunganley) [inline views/homepage.html]: move this section to a footer.html file and import it here. this way we can reuse on other pages

File: views/homepage.html [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+    .Gdansk-contact {
+        float: right;
+        padding-right: 100px;
+
+    }
+</style>
+
+<body class="body">
+    <div class="nav">
```
Comment (shaunganley) [inline views/homepage.html]: move this section to a heading.html file and import. that way we can reuse on other pages

**Review by shaunganley — CHANGES_REQUESTED** [review]

**Review by shaunganley — APPROVED** [review]

**Pull request merged**